hoot-services/src/main/java/hoot/services/controllers/grail/DeriveChangesetReplacementCommand.java

```diff
@@ -54,8 +63,35 @@
         substitutionMap.put("HOOT_OPTIONS", hootOptions);
         substitutionMap.put("DEBUG_LEVEL", debugLevel);
 
-        String command = "hoot changeset-derive-replacement --${DEBUG_LEVEL} -C DeriveChangeset.conf ${HOOT_OPTIONS} ${INPUT1} ${INPUT2} ${BOUNDS} ${OSC_FILE} --full-replacement --stats";
+        String command = "hoot changeset-derive-replacement --${DEBUG_LEVEL} -C DeriveChangeset.conf ${HOOT_OPTIONS} ${INPUT1} ${INPUT2} ${BOUNDS} ${OSC_FILE} --stats";
```

Comment:
these changes make the alpha distance adaptive to the extent of the data (default is 1000 meters but can result in an `alpha shape area is zero` error if the value is too large.

Also remove the `--full-replacement` param so only features inside the alpha shape are removed from the reference layer.   This will be important when the reference layer clip is buffered a bit to allow features outside the AOI to be loaded by Hoot so that road snapping may occur.